node-server/src/services/synchronizer.js

```diff
@@ -8,12 +8,10 @@ class Synchronizer {
   }
 
   sync = async () => {
-    let latestMessages = await this.db.getLastMessageIds()
-
+    const latestMessages = await this.db.getLastMessageIds()
     let latestByNodeId = {}
     latestMessages.forEach(obj => { latestByNodeId[obj.node_id] = obj['MAX(id)'] })
```

Comment:
I think this should be like

```suggestion
    latestMessages.forEach(obj => { latestByNodeId[obj.nodeId] = obj.id })
```